src/pycram/designators/location_designator.py

```diff
@@ -344,46 +361,32 @@ def __iter__(self) -> Location:
             container_joint: self.handle.world_object.get_joint_limits(container_joint)[1] / 1.5},
                                                self.handle.name)
 
-        ground_pose = Pose(self.handle.part_pose.position_as_list())
-        ground_pose.position.z = 0
-        occupancy = OccupancyCostmap(distance_to_obstacle=0.25, from_ros=False, size=200, resolution=0.02,
-                                     origin=ground_pose)
-        gaussian = GaussianCostmap(200, 15, 0.02, ground_pose)
-
-        final_map = occupancy + gaussian
         joint_type = self.handle.world_object.joints[container_joint].type
+        final_map = self.setup_costmaps()
         if joint_type == JointType.PRISMATIC:
             self.adjust_map_for_drawer_opening(final_map, init_pose, goal_pose)
 
+        target_sequence = [init_pose, half_pose, goal_pose]
```

Comment:
half_pose shouldn't be in the target sequence, it is only used for calculating the rotation